packages/monitoring-experience-api/src/routes.ts

```diff
@@ -1,11 +1,14 @@
 import { Router } from 'express'
 import { ElasticSearchHealthController } from './api/elasticSearchHealthController'
 import { Health } from './api/health'
+import { withIpWhitelistValidation } from './middleware/ipWhitelistValidation'
 
 const router = Router()
 
-const elasticSearchHealthController = new ElasticSearchHealthController()
-const healthController = new Health()
+const elasticSearchHealthController = new (withIpWhitelistValidation(
+  ElasticSearchHealthController
+))()
+const healthController = new (withIpWhitelistValidation(Health))()
```

Comment:
Do not add whitelist validation to this health endpoint, only to elasticsearch health

Reply:
Sure, I will remove it from the health.